### PR TITLE
Multiple advertising sets sample

### PR DESCRIPTION
**Pull request by KAGA164**
This Pr introduces the Multiple advertising sets sample.
This example starts two advertising sets:

- non-connectable advertising with the Nordic website URI data and Nordic Beacon name
- connectable advertising with the DIS UUID and Nordic multi adv sets name

### CODE REVIEW

File: samples/bluetooth/multiple_adv_sets/README.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+After programming the sample to your dongle or development kit, test it by performing the following steps:
+
+1. |connect_terminal_specific|
+#. Reset the kit.
+#. Start the `nRF Connect for Mobile`_ application on your smartphone or tablet.
+#. The device is advertising as ``Nordic multi adv sets`` and ``Nordic Beacon``.
+#. Choose the connectable variant and connect to the device from the nRF Connect application.
+#. On the scanner tab check if ``Nordic Beacon`` still advertises.
+   You can use it to open the Nordic website.
```
Comment (peknis) [inline samples/bluetooth/multiple_adv_sets/README.rst:69]:
```suggestion
#. On the :guilabel:`scanner` tab, check if ``Nordic Beacon`` still advertises.

   You can use it to open the nRF Beacon website.
   
```

File: samples/bluetooth/multiple_adv_sets/README.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+#. Disconnect from the device.
+#. In the scanner tab check again if there is advertiser with the ``Nordic multi adv sets`` name.
```
Comment (peknis) [inline samples/bluetooth/multiple_adv_sets/README.rst:71]:
```suggestion
#. In the :guilabel:`scanner` tab, check again if there is an advertiser with the name ``Nordic multi adv sets``.
```

File: samples/bluetooth/multiple_adv_sets/README.rst [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+   :depth: 2
+
+The Multiple advertising sets sample demonstrates how to use the Bluetooth® advertising sets.
+
+Overview
+********
+
+The sample implements two advertising sets:
+  * Non-connectable advertising with the URI to the https://www.nordicsemi.com website in the advertising data.
+    It shows the Bluetooth® Broadcaster role (*Beacon*) functionality.
```
Comment (peknis) [inline samples/bluetooth/multiple_adv_sets/README.rst:17]:
```suggestion
    It shows the Bluetooth Broadcaster role (*Beacon*) functionality.
```
You only need the trademark sign for the first instance of Bluetooth.

File: samples/bluetooth/multiple_adv_sets/README.rst [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+  * Connectable advertising with the 16-bit UUID of the Device Information GATT Service in the advertising data.
+    It show the Bluetooth® Peripheral role functionality.
```
Comment (peknis) [inline samples/bluetooth/multiple_adv_sets/README.rst:19]:
```suggestion
    It shows the Bluetooth Peripheral role functionality.
```

File: samples/bluetooth/multiple_adv_sets/README.rst [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+   :local:
+   :depth: 2
+
+The Multiple advertising sets sample demonstrates how to use the Bluetooth® advertising sets.
+
+Overview
+********
+
+The sample implements two advertising sets:
+  * Non-connectable advertising with the URI to the https://www.nordicsemi.com website in the advertising data.
```
Comment (peknis) [inline samples/bluetooth/multiple_adv_sets/README.rst:16]:
```suggestion
The sample implements two advertising sets:

  * Non-connectable advertising with the URI to the https://www.nordicsemi.com website in the advertising data.
```

File: samples/bluetooth/multiple_adv_sets/README.rst [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+    It shows the Bluetooth® Broadcaster role (*Beacon*) functionality.
+  * Connectable advertising with the 16-bit UUID of the Device Information GATT Service in the advertising data.
+    It show the Bluetooth® Peripheral role functionality.
+
+When started, the sample creates two advertising sets.
+You can use the scanner device to check it.
+On your scanner device you can observe two advertisers.
```
Comment (peknis) [inline samples/bluetooth/multiple_adv_sets/README.rst:23]:
```suggestion
Use your scanner device to observe the two advertisers:

```

File: samples/bluetooth/multiple_adv_sets/README.rst [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+Overview
+********
+
+The sample implements two advertising sets:
+  * Non-connectable advertising with the URI to the https://www.nordicsemi.com website in the advertising data.
+    It shows the Bluetooth® Broadcaster role (*Beacon*) functionality.
+  * Connectable advertising with the 16-bit UUID of the Device Information GATT Service in the advertising data.
+    It show the Bluetooth® Peripheral role functionality.
+
+When started, the sample creates two advertising sets.
```
Comment (peknis) [inline samples/bluetooth/multiple_adv_sets/README.rst:21]:
```suggestion
When you start the sample, it creates two advertising sets.
```

File: samples/bluetooth/multiple_adv_sets/README.rst [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+You can use the scanner device to check it.
+On your scanner device you can observe two advertisers.
+The first one with the ``Nordic Beacon`` name.
+You can use it to open the Nordic website.
+The second one with the ``Nordic multi adv sets`` name.
+You can use it to establish connection with the device.
+Use your central device to establish the connection with this sample.
+When connected, the non-connectable advertiser will be still presented on your scanner device.
+After disconnection the connectable advertising starts again.
```
Comment (peknis) [inline samples/bluetooth/multiple_adv_sets/README.rst:30]:
```suggestion
When connected, the non-connectable advertiser is still presented on your scanner device.
After disconnection, the connectable advertising starts again.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by grochu — COMMENTED** [review]
Looks good, just 1 minor comment.

File: samples/bluetooth/multiple_adv_sets/README.rst [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+************
+
+This sample uses the following |NCS| libraries:
+
+* :ref:`dk_buttons_and_leds_readme`
+
+In addition, it uses the following Zephyr libraries:
+
+* ``include/kernel.h``
+
```
[the remark this answers is not in the document]
Comment (KAGA164) [reply samples/bluetooth/multiple_adv_sets/README.rst:87]: Can be. I added it, however the DIS has empty header file

File: samples/bluetooth/multiple_adv_sets/README.rst [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+On your scanner device you can observe two advertisers.
+The first one with the ``Nordic Beacon`` name.
+You can use it to open the Nordic website.
+The second one with the ``Nordic multi adv sets`` name.
+You can use it to establish connection with the device.
+Use your central device to establish the connection with this sample.
+When connected, the non-connectable advertiser will be still presented on your scanner device.
+After disconnection the connectable advertising starts again.
+
+Requirements
```
[the remark this answers is not in the document]
Comment (KAGA164) [reply samples/bluetooth/multiple_adv_sets/README.rst:32]: Done

File: samples/bluetooth/multiple_adv_sets/README.rst [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+Testing
+=======
+
+After programming the sample to your dongle or development kit, test it by performing the following steps:
+
+1. |connect_terminal_specific|
+#. Reset the kit.
+#. Start the `nRF Connect for Mobile`_ application on your smartphone or tablet.
+#. The device is advertising as ``Nordic multi adv sets`` and ``Nordic Beacon``.
+#. Choose the connectable variant and connect to the device from the nRF Connect application.
```
[the remark this answers is not in the document]
Comment (KAGA164) [reply samples/bluetooth/multiple_adv_sets/README.rst]: Ok

File: samples/bluetooth/multiple_adv_sets/README.rst (hunk 1 above)
Comment (KAGA164) [reply samples/bluetooth/multiple_adv_sets/README.rst]: Done

File: samples/bluetooth/multiple_adv_sets/README.rst (hunk 2 above)
Comment (KAGA164) [reply samples/bluetooth/multiple_adv_sets/README.rst]: Done

File: samples/bluetooth/multiple_adv_sets/README.rst (hunk 3 above)
Comment (KAGA164) [reply samples/bluetooth/multiple_adv_sets/README.rst]: Ok, done

File: samples/bluetooth/multiple_adv_sets/README.rst (hunk 4 above)
Comment (KAGA164) [reply samples/bluetooth/multiple_adv_sets/README.rst]: Done

File: samples/bluetooth/multiple_adv_sets/README.rst (hunk 6 above)
Comment (KAGA164) [reply samples/bluetooth/multiple_adv_sets/README.rst]: Done

File: samples/bluetooth/multiple_adv_sets/README.rst [hunk 12]
```diff
@@ -0,0 +1,88 @@
...
+  * Connectable advertising with the 16-bit UUID of the Device Information GATT Service in the advertising data.
+    It show the Bluetooth® Peripheral role functionality.
+
+When started, the sample creates two advertising sets.
+You can use the scanner device to check it.
+On your scanner device you can observe two advertisers.
+The first one with the ``Nordic Beacon`` name.
+You can use it to open the Nordic website.
+The second one with the ``Nordic multi adv sets`` name.
+You can use it to establish connection with the device.
```
[the remark this answers is not in the document]
Comment (KAGA164) [reply samples/bluetooth/multiple_adv_sets/README.rst]: Ok

File: samples/bluetooth/multiple_adv_sets/README.rst (hunk 7 above)
Comment (KAGA164) [reply samples/bluetooth/multiple_adv_sets/README.rst]: Done

File: samples/bluetooth/multiple_adv_sets/README.rst (hunk 8 above)
Comment (KAGA164) [reply samples/bluetooth/multiple_adv_sets/README.rst]: Done

File: samples/bluetooth/multiple_adv_sets/README.rst (hunk 10 above)
[the remark this answers is not in the document]
Comment (peknis) [reply samples/bluetooth/multiple_adv_sets/README.rst:32]: I don't think you moved the Requirements section yet.

**Review by peknis — CHANGES_REQUESTED** [review]

File: samples/bluetooth/multiple_adv_sets/README.rst [hunk 13]
```diff
@@ -0,0 +1,93 @@
...
+After programming the sample to your dongle or development kit, test it by performing the following steps:
+
+1. |connect_terminal_specific|
+#. Reset the kit.
+#. Start the `nRF Connect for Mobile`_ application on your smartphone or tablet.
+
+   The device is advertising as ``Nordic multi adv sets`` and ``Nordic Beacon``.
+
+#. Select the connectable variant and connect to the device from the application.
+#. On the scanner tab check if ``Nordic Beacon`` still advertises.
```
Comment (peknis) [inline samples/bluetooth/multiple_adv_sets/README.rst:70]:
```suggestion
#. On the :guilabel:`scanner` tab check if ``Nordic Beacon`` still advertises.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by grochu — APPROVED** [review]

File: samples/bluetooth/multiple_adv_sets/README.rst (hunk 13 above)
Comment (KAGA164) [reply samples/bluetooth/multiple_adv_sets/README.rst]: Done

**Review by peknis — APPROVED** [review]

**Pull request merged**